entrypoint.sh

```diff
@@ -62,8 +62,6 @@ migrate () {
 	unset PGPASSWORD
 }
 
-PGDATA=${PGVOLUME}/data
```

Comment:
Couldn't find where `PGDATA` is read from in the new way.

Reply:
PGDATA is defined in the official base image.